addons/website_slides/controllers/main.py

```diff
@@ -301,14 +301,32 @@ def create_slide(self, *args, **post):
     # --------------------------------------------------
     @http.route('/slides/embed/<int:slide_id>', type='http', auth='public', website=True, sitemap=False)
     def slides_embed(self, slide_id, page="1", **kw):
+        """
+        Detect if the slide is embedded in a website outside of the Odoo database.
+        The detection is there to differenciate cases where the slide is present on a webpage that is not managed by the
+        odoo db (so not any of the domain, not the base_url, etc ... )
+
+        It is detected via the Referer HTTP header
+            If the referer is absent -> Not embedded
+            Then we look at the domains configured in the Website module : if their domain is inside the referer url -> Not embedded
+            We also look at the base_url of ir.config_parameter : same logic as above
+            And finally also at the the host of the httprequest : same logic as above
+        This is important because externally embedding the slide will display Share options inside the slide itself,
+        rather than on the bottom of the page in the Odoo slide module
+        This detection should focus on having the least amount of False Negative (detecting it's embedded when it's actually a Odoo DB)
```

Comment:
"when it's actually outside a Odoo DB" ?

Reply:
I can change it to `This detection should focus on having the least amount of False Negative (detecting the slide is embedded when it's actually shown on a page inside a domain of the Odoo DB)` if you prefer? I don't really know how to say the last part but I think it'll be clearer with that ?

Reply:
Oh you are explaining what a False Negative is, I just got it.
Hmm, what do you think about:
> This detection should focus on having the least amount of False Negative.
> In short, we want a slide to be considered as embedded when it's actually shown on website outside of this Odoo database.

Reply:
In the end the false negative part was useless so I explained it like this `        In case of uncertainty about the situation, we should assume it is NOT embedded`